developers.libra.org/COPYRIGHT.md

```diff
@@ -0,0 +1 @@
+COPYRIGHT.txt
```

Comment:
?

Reply:
I am just generally copying things over as-is from `website`. We can remove extraneous once we land this thing, imho.